packages/elements-core/src/components/Docs/Docs.tsx

```diff
@@ -24,6 +24,11 @@ interface BaseDocsProps {
    * URI of the document
    */
   uri?: string;
+
+  /**
+   * Shows only operation document without right column
+   */
+  docsOnly?: boolean;
```

Comment:
After careful consideration I would go for having 2 options here: `hideTryIt` and `hideTryItPanel`.